skimage/morphology/binary.py

```diff
@@ -169,6 +223,13 @@ def binary_opening(image, footprint=None, out=None):
     out : ndarray of bool, optional
         The array to store the result of the morphology. If None
         is passed, a new array will be allocated.
+    border_value : bool or None, optional
+        Value at the border in the output array. ``None`` is ``True`` for the
+        erosion and ``False`` for the dilation, causing pixels outside the
+        image not to influence the results. Default is ``None``.
```

Comment:
This would allow setting `[True, True]`, `[False, False]`, `[True, False]` and `[False, True]`. Three of those options correspond to `True`, `False` and `None` in the current iteration, the fourth option doesn't seem especially interesting to me. It makes the parameter more complex, I'm not sure the added option is worth that complexity.

What we could do however is change this for a `mode` parameter like in the gray-scale case, with options "max", "min" and "ignore".

Reply:
Sounds good and seems more consistent to to have a `mode` parameter everywhere. :+1: